analyzers/tests/SonarAnalyzer.UnitTest/TestCases/PointersShouldBePrivate.CSharp9.cs

```diff
@@ -1,29 +1,31 @@
 ﻿using System;
 
-unsafe public record Record
+public unsafe record Record
 {
     private IntPtr p1;
-    protected IntPtr p2; // Noncompliant
+    public UIntPtr p2; // Noncompliant
+}
+
+public unsafe class Class
+{
+    private IntPtr p1;
+    protected int* p2; // Noncompliant
     protected readonly IntPtr p3;
     public IntPtr p4; // Noncompliant
 
     private UIntPtr p5;
-    protected UIntPtr p6; // Noncompliant
```

Comment:
why replace this test with the `char*` one? 
(I'm not suggesting to change it, just trying to understand)

Reply:
IntPtr case is already covered with field p4, and just wanted to add a test case with the pointer types.